lib/tests/helpers.test.js

```diff
@@ -161,19 +161,21 @@ describe("Submit and Template helpers", () => {
     );
   });
   test("getFormByID returns form when ID is specified", async () => {
-    mockedAxios.mockResolvedValue({
-      status: 200,
-      data: {
-        data: [
-          {
-            formID: "1",
-            elements: { test: "test" },
-            publishingStatus: true,
-            securityAttribute: "Unclassified",
-          },
-        ],
-      },
-    });
+    mockedAxios.mockImplementationOnce(() =>
```

Comment:
There's no issue using `mockResolvedValue`. Also, this PR shouldn't touch the template API at all. Either way, an update of this section by using mockResolveValue instead of mockImplementationOnce will be added.